### ISSUE TITLE
Rule: Multiline blocks should be enclosed in curly braces

### ISSUE THREAD

**Issue by fmallet** [SonarSource/SonarTS#66]
Implements [RSPEC-2681](https://jira.sonarsource.com/browse/RSPEC-2681)

**Comment by inverno** [comment]
This is raised by the "align" rule in tslint : https://palantir.github.io/tslint/rules/align/

It might be more interesting to catch this case :
```
if(x) doSomething(); doSomething();
```

**Comment by fmallet** [comment]
Even if there is an overlap between the two rules @inverno, one is an efficient bug detection rule raising few issues but all of them being relevant whereas the tslint rule is a noisy coding style rule. So having both of them would make sense to me.
